### DISCUSSION TITLE
Display small status bar on all pages

### DISCUSSION THREAD

**Discussion by MaTToX3** [PrismLibrary/Prism#2306] [Xamarin.Forms]
I would like to display a small status bar at the bottom of my app on each of the pages I have but I don't want to do it manually on every page - it would have to be global. Is it possible to do it in that way? Was looking around and could not find any info.

Thanks in advance!

**Comment by Yerodin84** [comment]
Yes, its possible. 

Looking for Composite Commands
You need e Pub Sub Event Class 
Your Pages have RegionBaseViewModel as Baseclass then you publish their the Event what you want and subscribe it in your Viewmodelclass.
In Your ViewModel you define e Method where you set e the result of the method to a Property
That's all
